tests/Signal/subtle/watch-unwatch.test.ts

```diff
@@ -160,4 +160,34 @@ describe('watch and unwatch', () => {
     expect(w.getPending()).toStrictEqual([]);
     expect(d).toBe(4);
   });
+  it('can unwatch multiple signals', async () => {
+    const signals = [...Array(7)].map((_, i) => new Signal.State(i));
+    const notify = vi.fn();
+    const watcher = new Signal.subtle.Watcher(notify);
+    const expectSources = (expected: typeof signals) => {
+      const sources = Signal.subtle.introspectSources(watcher) as typeof signals;
+      sources.sort((a, b) => signals.indexOf(a) - signals.indexOf(b));
+      expected.sort((a, b) => signals.indexOf(a) - signals.indexOf(b));
+      return expect(sources).toEqual(expected);
+    };
+
+    watcher.watch(...signals);
+    expectSources(signals);
+
+    const unwatched = [0, 3, 4, 6].map((i) => signals[i]);
+    const watched = signals.filter((s) => !unwatched.includes(s));
+
+    watcher.unwatch(...unwatched);
+    expectSources(watched);
+
+    let expectedNotifyCalls = 0;
+    for (const signal of signals) {
+      signal.set(signal.get() + 1);
+      if (watched.includes(signal)) ++expectedNotifyCalls;
+
+      expect(notify).toHaveBeenCalledTimes(expectedNotifyCalls);
+
+      watcher.watch();
+    }
```

Comment:
i might just add `expect(expectedNotifyCalls).toBe(3)` at the end here just to be clear.

Reply:
How about `expect(notify).toHaveBeenCalledTimes(3)` directly?